### PR TITLE
22679-Metacello-attributes-should-be-based-on-SystemVersion-instead-of-hardcoding

### PR DESCRIPTION
**Pull request by jecisc**
Do not hardcode platform attributes of Metacello.

### CODE REVIEW

**Comment by jecisc** [comment]
Fixes https://pharo.fogbugz.com/f/cases/resolve/22679/Metacello-attributes-should-be-based-on-SystemVersion-instead-of-hardcoding

**Comment by pavel-krivanek** [comment]
Originally it was hardcoded to be able to load the projects designed for the older version at the beginning of the new Pharo version development. But yes, we should change it

**Review by akgrant43 — APPROVED** [review]
Looks good - reduces the number of changes required as part of change a version number and it answers the same array as previously in my Pharo 7 image.

File: src/System-Support/SmalltalkImage.class.st [hunk 1]
```diff
@@ -1128,9 +1128,14 @@ SmalltalkImage >> memoryHogs [
...
 { #category : #miscellaneous }
 SmalltalkImage >> metacelloPlatformAttributes [
-    "Returns the tags for the conditional platform loading in Metacello. Pay attention the order is important: from most  to least general." 
-    "For release integrators, we should not have #'pharo1.3x' **and** #'pharo1.4.x'"
-   ^ #(#squeakCommon #pharo #'pharo7.x' #'pharo7.0.x')
+	"Returns the tags for the conditional platform loading in Metacello. Pay attention the order is important: from most  to least general."
+
+	| attributes systemeVersion |
+	attributes := OrderedCollection withAll: #(#squeakCommon #pharo).
+	systemeVersion := SystemVersion current.
```
Comment (gcotelli) [inline src/System-Support/SmalltalkImage.class.st]: Tiny typo here: systemeVersion -> systemVersion

**Comment by estebanlm** [comment]
we will revert because is breaking the test process (we'll see why)

**Pull request merged**